book/website/collaboration/new-community/new-recruitment.md

```diff
@@ -0,0 +1,524 @@
+
+
+### Social media and outreach 
+
+Depending on your organisation’s social media policy, how you share the role online in your personal networks and brand accounts can also influence its reach. Sharing job postings in targeted LinkedIn groups (for example Diversity in Tech), and using targeted hashtags (for example #DiversityInTech, #WomenInAI) can give your advert a wider spread, reaching  candidates who may not otherwise encounter it.
+
+
+### Targeting external networks and organisations 
+
+
+Targeting external organisations and networks with a focus on underrepresented groups will broaden your reach. We have included some suggestions around this in the Resources section.
+
+
+## Checklist for recruitment 
+
+The checklist and timeline below outline a range of activities which can be undertaken to increase the diversity of your applicant pool. This list is not exhaustive so you will need to consult your organisation’s recruitment policies and recommendations. Having a specific timeline outlined and communicated from the start of the recruitment process is important not only for hiring teams but also for applicants. 
+
+
+### Before advertising 
+
+
+
+- [ ] Ensure effective communication  between hiring manager and recruitment team - know what your recruitment team is already doing (for example  where roles will be advertised) to avoid working in silos
+- [ ] Writing the job description (JD)
+    * Detail what the person will be doing - recommend 10-12 bullet points
+    * Write in plain English, removing any jargon and abbreviations
+    * Consider unconscious bias against all characteristics - often unconscious bias moves beyond the nine protected characteristics and into dimensions such as education level, socioeconomic status, geography, and hobbies.
+        * Avoid superlatives
+        * Check for accessibility of advert using a screen reader
+        * Keep mandatory skills to a minimum and thoroughly consider whether all criteria are essential
+        * Avoid asking for mandatory travel, weekend work and overtime, if possible
+    * Remove all biased language in the job advert
+        * Consider using a gender decoder to identify words/phrases that may imply a specific gender and re-phrase if needed
+    * Provide clear guidelines & deadlines plus instructions & contacts for people requiring reasonable adjustments 
```

Comment:
Also a contact for general inquiries / people who want to check "am I suited for the job/is this job suited for me"

Reply:
I would also update this to to "consider stating what accessibility factors have already been taken into consideration, such as live captions or sending questions in advance."

Doing something like this means that potential applicants see your commitment to diversity and accessibility from the  beginning, and it may make them feel more confident about asking for accommodations they specifically need.